db/migrate/20190723094834_add_index_to_podcasts_episodes.rb

```diff
@@ -0,0 +1,5 @@
+class AddIndexToPodcastsEpisodes < ActiveRecord::Migration[5.2]
+  def change
+    add_index :podcast_episodes, :podcast_id
```

Comment:
you might want to add this concurrently, I don't know how big the production table is but still...

Reply:
Agree, I had a feeling that I forgot something, thanks!